resources/views/campaigns/roles/_form.blade.php

```diff
@@ -2,5 +2,17 @@
 <div class="field-name required">
     <label>{{ trans('campaigns.roles.fields.name') }}</label>
     {!! Form::text('name', null, ['placeholder' => trans('campaigns.roles.placeholders.name'), 'class' => 'form-control', 'maxlength' => 45, 'required']) !!}
+    @if (isset($roleId))
+        <div class="field-skip-zero checkbox">
```

Comment:
`field-skip-zero`? :) We give each field it's own css class name to allow people to style it. Should be `field-duplicate`